.travis.yml

```diff
@@ -359,11 +360,50 @@ matrix:
         - cmake -DBUILD_SHARED_LIBS=On ../libs/atomic/test/test_cmake
         - cmake --build .
 
+# CMake test from the Boost superproject
+    - os: linux
+      env: TEST_CMAKE=1 CMAKE_BOOST=1
+      script:
+        - mkdir __build_static__ && cd __build_static__
+        - cmake -DBOOST_ENABLE_CMAKE=1 -DBoost_VERBOSE=1 -DBOOST_INCLUDE_LIBRARIES=atomic ..
+        - cmake --build .
+        - ctest --output-on-failure -R boost_atomic
+        - cd ..
+        - mkdir __build_shared__ && cd __build_shared__
+        - cmake -DBOOST_ENABLE_CMAKE=1 -DBoost_VERBOSE=1 -DBOOST_INCLUDE_LIBRARIES=atomic -DBUILD_SHARED_LIBS=ON ..
+        - cmake --build .
+        - ctest --output-on-failure -R boost_atomic
+
+# CMake install test
+    - os: linux
+      env: TEST_CMAKE=1 CMAKE_INSTALL=1
+      script:
+        - mkdir __build__ && cd __build__
+        - cmake -DBOOST_ENABLE_CMAKE=1 -DBoost_VERBOSE=1 -DBOOST_INCLUDE_LIBRARIES="atomic;assert;config;static_assert;type_traits" -DCMAKE_INSTALL_PREFIX=~/.local ..
```

Comment:
I don't like that there is a list of dependencies. Also, install prefix of `~/.local` does not feel right.